integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install

```diff
@@ -1,39 +1,27 @@
 set -e
 
 sudo yum install -y curl java
-sudo mkdir -p /opt/tomcat/stage
-sudo curl "https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.58/bin/apache-tomcat-9.0.58.tar.gz" -o /opt/tomcat/stage/tomcat.tgz
-sudo tar -xvzf /opt/tomcat/stage/tomcat.tgz -C /opt/tomcat --strip 1
-
-cat << EOF > tomcat.service
+sudo yum install tomcat -y
+sudo cat << EOF | sudo tee /etc/systemd/system/tomcat.service
```

Comment:
At least this is in /etc instead of /usr, but please use a systemd drop-in and just customize the one thing you need (the environment variables?). See https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.14.3

Reply:
This one seems obsolete now. It later turned out that the only available package is tomcat7 on CentOS, which we don't support. So this change has been reverted.